src/main/java/frc/team2412/robot/commands/launcher/SetLaunchSpeedCommand.java

```diff
@@ -11,7 +11,6 @@ public class SetLaunchSpeedCommand extends Command {
 	public SetLaunchSpeedCommand(LauncherSubsystem launcherSubsystem, double speed) {
 		launcherSpeed = speed;
 		this.launcherSubsystem = launcherSubsystem;
-		addRequirements(launcherSubsystem);
```

Comment:
Why was this requirement removed?

Reply:
It was to address James' comment that parallel command groups cannot have two requirements for the same subsystem. Since SetLaunchSpeedCommand ends immediately I thought it was in less need of requiring LauncherSubsystem

Reply:
Composing multiple commands in parallel that use the same subsystem isn't going to work. Rather than removing the requirement (which means sequential use elsewhere is likely going to end up with two commands telling a subsystem to do different things), it'd be better to add a new command that does the combination that's needed. To address the proliferation of tiny command classes, consider adding command helpers to the subsystem (like `DrivebaseSubsystem::rotateToAngle` did) rather than small files like this.
From https://github.com/wpilibsuite/allwpilib/blob/6afff996401343c3bfcc46a40a40abda429e5077/wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/ParallelCommandGroup.java#L51C1-L54C8
      if (!Collections.disjoint(command.getRequirements(), m_requirements)) {
        throw new IllegalArgumentException(
            "Multiple commands in a parallel composition cannot require the same subsystems");
      }

Reply:
@iamawesomecat Can you address this in your PR? I think this fix would be getting away from the original purpose of this PR. I am going to remove my current fix from this PR.

Reply:
This was fixed in my PR as far as im aware.